src/components/include/utils/shared_ptr.h

```diff
@@ -38,8 +38,11 @@
 #include <stdint.h>
 
 #include "utils/macro.h"
+#ifdef QT_PORT
+#include <QAtomicPointer>
```

Comment:
see dd6eb9b<https://github.com/nk0leg/sdl_core_winport/commit/dd6eb9b5778fcc9fe2005ba1fd5166d74c060392>


________________________________

This e-mail and any attachment(s) are intended only for the recipient(s) named above and others who have been specifically authorized to receive them. They may contain confidential information. If you are not the intended recipient, please do not read this email or its attachment(s). Furthermore, you are hereby notified that any dissemination, distribution or copying of this e-mail and any attachment(s) is strictly prohibited. If you have received this e-mail in error, please immediately notify the sender by replying to this e-mail and then delete this e-mail and any attachment(s) or copies thereof from your system. Thank you.